Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/OnboardingData.kt

```diff
@@ -0,0 +1,48 @@
+package de.rki.coronawarnapp.storage
+
+import android.content.Context
+import de.rki.coronawarnapp.util.di.AppContext
+import de.rki.coronawarnapp.util.preferences.clearAndNotify
+import de.rki.coronawarnapp.util.preferences.createFlowPreference
+import org.joda.time.Instant
+import javax.inject.Inject
+
+class OnboardingData @Inject constructor(
```

Comment:
Where does the migration happen?

Reply:
Nowhere yet, i still want to align with @jurajkusner on the specifics of how to migrate

Reply:
Migration will be done for all preferences in a seperate PR